tests/languages/java-file/Makefile

```diff
@@ -0,0 +1,35 @@
+include ../../common.mk
+
+PROG=MainApp
+MAINAPP_JAVA=app/MainApp.java
+FILEREADER_JAVA=app/FileReadSample.java
+
+DISK_IMAGE=sgxlkl-java.img
+IMAGE_SIZE=400M
+
+ifeq ($(SGXLKL_VERBOSE),)
+SGXLKL_ENV=\
+   SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=1 SGXLKL_TRACE_LKL_SYSCALL=0 SGXLKL_TRACE_SIGNAL=0
+else
+SGXLKL_ENV=
+endif
+
+.DELETE_ON_ERROR:
+.PHONY: run run-hw run-sw clean
+
+$(DISK_IMAGE): $(MAINAPP_JAVA) $(FILEREADER_JAVA)
+	docker build -t java-app:dev .
```

Comment:
This works, but in tests we should use the Dockerfile directly in sgx-lkl-disk to avoid creating tagged/named Docker images on the CI or dev machines. You can use `--docker=Dockerfile` and remove the docker command.

Reply:
Done